src/components/picker/index.tsx

```diff
@@ -242,11 +254,6 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
         mode={mode}
         useDialog={useDialog}
         items={useItems ? items : undefined}
-        topBarProps={{
```

Comment:
remove 'topBarProps' from the PickerItemsList  types as well

Reply:
about that, I notice 1 issue when using `WheelPicker`.
when using wheel picker the `renderCancel` function getting the `cancelButtonProps || cancelLabel` from the `topBarProps`.
And also `renderPickerHeader` uses `topBarProps` to get the `onDone, onCancel` functionalities.
Let's sync about that

Reply:
I think the problem is that the WheelPicker has it own "header" instead of using the Dialog's header...